test/testenv/verificationutils.go

```diff
@@ -569,3 +569,38 @@ func VerifyAppFrameworkDeployment(standalone *enterprisev1.Standalone) {
 	}
 
 }
+
+// VerifyAppInstall verify that app of specific version is installed
+func VerifyAppInstall(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps map[string]string, versionCheck bool, statusCheck string) {
+	for appName, version := range apps {
+		status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
+		logf.Log.Info("App info returned for app", "App-name", appName, "status", status, "versionInstalled", versionInstalled)
+		gomega.Expect(err).To(gomega.Succeed(), "Unable to get app status on pod ")
+		gomega.Expect(strings.ToLower(status)).Should(gomega.Equal(strings.ToLower(statusCheck)))
```

Comment:
Add logs before the checks for help with debugging.

Reply:
logs added in line just after status and version are retrieved.